apmpackage/model.go

```diff
@@ -0,0 +1,77 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package apmpackage
+
+type FieldDefinition struct {
+	Name        string `yaml:"name,omitempty"`
+	Key         string `yaml:"key,omitempty"`
+	Title       string `yaml:"title,omitempty"`
+	Group       *int   `yaml:"group,omitempty"`
+	Level       string `yaml:"level,omitempty"`
+	Required    *bool  `yaml:"required,omitempty"`
+	Type        string `yaml:"type,omitempty"`
+	Format      string `yaml:"format,omitempty"`
+	Description string `yaml:"description,omitempty"`
+	Release     string `yaml:"release,omitempty"`
+	Alias       string `yaml:"alias,omitempty"`
+	Path        string `yaml:"path,omitempty"`
+	Footnote    string `yaml:"footnote,omitempty"`
+	// Example is not consistent in ECS schema (either single field or array)
+	//Example     string             `yaml:"example,omitempty"`
```

Comment:
Is there anything left to do with `Example`? Should it be an `interface{}`?

Reply:
This was copied over from the beats migration script, that probably can be removed, yes